packages/sources/finage-test/src/endpoint/http/uk-etf.ts

```diff
@@ -0,0 +1,61 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EquitiesEndpointTypes } from '../types'
+
+interface ResponseSchema {
+  symbol: string
+  price: number
+  timestamp: number
+  error?: string
+}
+
+type HttpTransportTypes = EquitiesEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      const symbol = `${param.base}`.toUpperCase()
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: `/last/etf/${symbol}`,
+          params: { apikey: config.API_KEY },
+        },
+      }
+    })
+  },
+  parseResponse: (params, res) => {
+    if (res.data.error) {
+      return params.map((param) => {
+        return {
+          params: param,
+          response: {
+            errorMessage:
+              'Could not retrieve valid data from Data Provider. This is likely an issue with the Data Provider or the input params/overrides',
```

Comment:
Minor: Including that this error occurred when calling the Finage ETF endpoint will make future debugging easier